ITISFoundation/osparc-simcore
# 🐛 Failing to start study

## What do these changes do?

Fixes an issue where a study would fail to start or fail to load the workbench and stay on the dashboard while running in the background. The issue seems to mainly affect Safari users with a new session.






## Related issue/s

- fixes https://github.com/ITISFoundation/osparc-simcore/issues/5259




## How to test

Open a private browser window in Safari and login
Start a new Classic TI planning

If this commit is successful  the study should open as expected.
(Basically the TI test should start having a better success rate)




## Dev-ops checklist
